### ISSUE TITLE
Access key/ Help text is not defined for 'Open' menu item

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#347]
### Steps to Reproduce

1. Launch Xamarin Workbook application.
2. Navigate to File menu item present in menu bar and Expand it
3. Navigate to 'Open' menu item and Observe Inspect properties

### Expected Behavior
Access key or Help text should be defined as 'CTRL+O' for 'Open' menu item

### Actual Behavior
Access key or Help text is not defined for 'Open' menu item
**Narrator Behavior:** Narrator is announcing Open menu item as 'open, menu item'
**NVDA Behavior:** NVDA is announcing Open menu item as 'open'
**Note:** Same behavior is also observed for 'Run All, Save, New ' menu items

### User Impact
Users who rely on screen reader will not be able to know the shortcut key for 'Open' menu item

### Screenshots
![open_menu_item_inspect](https://user-images.githubusercontent.com/22956130/40723168-ddebb574-643b-11e8-9bff-edf86f6f535d.png)
![open_menu_item_narrator](https://user-images.githubusercontent.com/22956130/40723189-ef70d086-643b-11e8-9ae6-772bc1f12236.png)
![open_menu_item_nvda](https://user-images.githubusercontent.com/22956130/40723196-f265f33e-643b-11e8-9d2a-dc95798e88bf.png)
![access_key_new_inspect](https://user-images.githubusercontent.com/22956130/40723220-0606b8b0-643c-11e8-8dd1-a10d884f0a7b.png)
![access_key_runall_inspect](https://user-images.githubusercontent.com/22956130/40723230-0990de2a-643c-11e8-8fba-ce19eae5c17d.png)
![access_key_save_inspect](https://user-images.githubusercontent.com/22956130/40723236-0cafc148-643c-11e8-8307-86a0aead7241.png)

### Environment
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)
NVDA Version: 2018.1

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.2.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
